src/platform/silabs/multi-ota/OTATlvProcessor.h

```diff
@@ -100,8 +100,19 @@ class OTATlvProcessor
     void SetWasSelected(bool selected) { mWasSelected = selected; }
     bool WasSelected() { return mWasSelected; }
 #ifdef SL_MATTER_ENABLE_OTA_ENCRYPTION
+    bool IsLastBlock() const { return mLastBlock; }
```

Comment:
The mLength flag seems superfluous:
```suggestion
    bool IsLastBlock() const { return mProcessedLength >= mLength; }
```